snuba/datasets/configuration/replays/storages/replays.yaml

```diff
@@ -8,6 +8,11 @@ schema:
   columns:
     [
       { name: replay_id, type: UUID },
+      {
```

Comment:
The storage change should be in the other PR. If you deploy this, your consumer might break until the migration is run.